getting-started/windows-calico/demo.md

```diff
@@ -0,0 +1,267 @@
+
+## Prerequisites
+
+To run this demo, you will need a [{{site.prodnameWindows}} cluster]({{site.baseurl}}/getting-started/windows-calico/quickstart) with
+Windows Server 1809 (build 10.0.17763). More recent versions of Windows Server may be used with a change to the demo manifests.
+
+>**Note**: Windows Server 1809 (build 10.0.17763) does not currently support [direct server return](https://techcommunity.microsoft.com/t5/networking-blog/direct-server-return-dsr-in-a-nutshell/ba-p/693710). This means that policy support is limited to only pod IP addresses.
+{: .alert .alert-info}
+
+You will also need [calicoctl]({{site.baseurl}}/getting-started/clis/calicoctl/install) installed and configured.
+
+## Create pods on Linux nodes
+
+First, create a client (busybox) and server (nginx) pod on the Linux nodes:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: calico-demo
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: busybox
+  name: busybox
+  namespace: calico-demo
+spec:
+  containers:
+  - args:
+    - /bin/sh
+    - -c
+    - sleep 360000
+    image: busybox:1.28
+    imagePullPolicy: Always
+    name: busybox
+  nodeSelector:
+    beta.kubernetes.io/os: linux
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: nginx
+  name: nginx
+  namespace: calico-demo
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.8
+    ports:
+    - containerPort: 80
+  nodeSelector:
+    beta.kubernetes.io/os: linux
+EOF
+```
+
+## Create pods on Window nodes
+
+Next we'll create a client (powershell) and server (porter) pod on the Windows nodes. First the create the powershell pod.
+
+>**Note**: The powershell and porter pod manifests below use images based on `mcr.microsoft.com/windows/servercore:1809`.
+If you are using a more recent Windows Server version, update the manifests to use a [servercore image](https://hub.docker.com/_/microsoft-windows-servercore)
+that matches your Windows Server version.
+{: .alert .alert-info}
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pwsh
+  namespace: calico-demo
+  labels:
+    app: pwsh
+spec:
+  containers:
+  - name: pwsh
+    image: mcr.microsoft.com/windows/servercore:1809
+    args:
+    - powershell.exe
+    - -Command
+    - "Start-Sleep 360000"
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/os: windows
+EOF
+```
+
+Next we'll create the porter server pod:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: porter
+  namespace: calico-demo
+  labels:
+    app: porter
+spec:
+  containers:
+  - name: porter
+    image: songtjiang/porter:1809
```

Comment:
We need to replace the image or push it to `calico` docker hub repository.

Reply:
Fixed